sdk/python/kfmd/metadata.py

```diff
@@ -171,11 +213,45 @@ def __init__(self,
     self.model_type = model_type
     self.training_framework = training_framework
     self.hyperparameters = hyperparameters
-    self.query = query
     self.labels = labels
     self._id = ""
     self._create_time = ""
 
+  def type_name(self):
+    return "artifact_types/kubeflow.org/alpha/model"
```

Comment:
Quick question: are these core types created in the backend during the initialization or inserted by the SDK?

Reply:
These types are created by the backend when server starts.